Ray.jl/test/runtime.jl

```diff
@@ -78,5 +78,41 @@
             @test !contains(stderr_logs, "Constructing CoreWorkerProcess")
         end
     end
+end
+
+@testset "serialize_args" begin
+    ray_config = ray_jll.RayConfigInstance()
+    put_threshold = ray_jll.max_direct_call_object_size(ray_config)
+    rpc_inline_threshold = ray_jll.task_rpc_inlined_bytes_limit(ray_config)
+    serialization_overhead = begin
+        sizeof(Ray.serialize_to_bytes(:_ => zeros(UInt8, put_threshold))) - put_threshold
+    end
+
+    @testset "put threshold" begin
+        a = :_ => zeros(UInt8, put_threshold - serialization_overhead)
+        b = :_ => zeros(UInt8, put_threshold - serialization_overhead + 1)
```

Comment:
was really scratching my head here until I remembered this is how we represent positional arguments.  thought it was some weird `fill` syntax.  maybe add a comment to clarify?

```suggestion
        # positional arguments to tasks have key `:_`
        a = :_ => zeros(UInt8, put_threshold - serialization_overhead)
        b = :_ => zeros(UInt8, put_threshold - serialization_overhead + 1)
```

Reply:
or use `flatten_args`

Reply:
actually, I think I like using `flatten_args` better here

Reply:
I'll add this further up as the first occurrence is by `serialization_overhead`